metloom/variables.py

```diff
@@ -73,6 +73,21 @@ class CdecStationVariables(VariableBase):
     TEMPAVG = SensorDescription("30", "AVG AIR TEMP", "TEMPERATURE, AIR AVERAGE")
     TEMPMIN = SensorDescription("32", "MIN AIR TEMP", "TEMPERATURE, AIR MINIMUM")
     TEMPMAX = SensorDescription("31", "MAX AIR TEMP", "TEMPERATURE, AIR MAXIMUM")
+    RH = SensorDescription("12", "Relative Humidity", "RELATIVE HUMIDITY")
+    # TODO confirm with CDWR if these depths are standard, no metadata available
+    TEMPGROUND = SensorDescription("52", "Ground Temperature Int", "Ground Temperature")
+    TEMPGROUND2 = SensorDescription(
+        "194", "GROUND TEMPERATURE -2", "GROUND TEMPERATURE OBS -2in"
+    )
+    TEMPGROUND4 = SensorDescription(
+        "195", "GROUND TEMPERATURE -4", "GROUND TEMPERATURE OBS -4in"
+    )
+    TEMPGROUND8 = SensorDescription(
+        "196", "GROUND TEMPERATURE -8", "GROUND TEMPERATURE OBS -8in"
+    )
+    SOLARRAD = SensorDescription("103", "Solar Radiation", "SOLAR RADIATION")
+    WINDSPEED = SensorDescription("9", "Wind Speed", "WIND SPEED")
+    WINDDIR = SensorDescription("10", "Wind Direction", "WIND DIRECTION")
```

Comment:
Can you change the SensorDescription.name (the second argument) to be all caps for these entries